demos/formula_recognition_demo/python/utils.py

```diff
@@ -157,14 +148,15 @@ class Status(Enum):
 
     def __init__(self, args, interactive_mode):
         self.args = args
-        log.info("Creating Inference Engine")
+        log.info('OpenVINO Inference Engine')
+        log.info('build: {}'.format(get_version()))
         self.ie = IECore()
-        self.ie.set_config(
-            {"PERF_COUNT": "YES" if self.args.perf_counts else "NO"}, args.device)
         self.encoder = read_net(self.args.m_encoder, self.ie)
         self.dec_step = read_net(self.args.m_decoder, self.ie)
         self.exec_net_encoder = self.ie.load_network(network=self.encoder, device_name=self.args.device)
+        log.info('Loaded model {} to {}'.format(args.m_encoder, args.device))
         self.exec_net_decoder = self.ie.load_network(network=self.dec_step, device_name=self.args.device)
+        log.info('Loaded model {} to {}'.format(args.m_decoder, args.device))
```

Comment:
In case more than one model, the log should contain the model types (here -`encoder model`, `decoder model`). It helps to distinguish similar logs

Reply:
Added model types to logs for all demos. (https://github.com/openvinotoolkit/open_model_zoo/pull/2567/commits/b6cab5ce3dda1402efc48c50d56509f06e0b5603)